atlas/packages/app/package.json

```diff
@@ -14,25 +14,60 @@
 	],
 	"repository": {
 		"type": "git",
-		"url": "https://github.com/Joystream/atlas.git"
+		"url": "https://github.com/Joystream/joystream.git"
 	},
 	"scripts": {
 		"start": "parcel public/index.html",
 		"build": "parcel build public/index.html",
 		"dev": "parcel public/index.html",
-		"test": "echo \"Error: run tests from root\" && exit 1"
+		"storybook": "start-storybook -p 6006 -c src/shared/.storybook",
+		"build-storybook": "build-storybook -c src/shared/.storybook",
+		"test": "echo \"Error: run tests from root\" && exit 1",
+		"chromatic": "chromatic --project-token=qq8aetz26u"
 	},
 	"bugs": {
-		"url": "https://github.com/Joystream/atlas/issues"
+		"url": "https://github.com/Joystream/joystream/issues"
 	},
 	"dependencies": {
 		"@emotion/core": "^10.0.28",
-		"@joystream/components": "1.0.0",
 		"@reach/router": "^1.3.3",
+		"@types/reach__router": "^1.3.5",
+		"@types/react": "^16.9.36",
+		"@types/react-dom": "^16.9.8",
+		"@types/react-redux": "^7.1.9",
+		"@types/redux": "^3.6.0",
 		"emotion-normalize": "^10.1.0",
 		"react": "^16.13.1",
 		"react-dom": "^16.13.1",
 		"react-redux": "^7.2.0",
-		"redux": "^4.0.5"
+		"redux": "^4.0.5",
+		"react-player": "^2.2.0",
+		"react-spring": "^8.0.27",
+		"use-resize-observer": "^6.1.0",
+		"chromatic": "^4.0.3",
+		"react-docgen-typescript-loader": "^3.7.1",
+		"storybook-addon-jsx": "^7.1.15",
+		"parcel": "^2.0.0-beta.1",
+		"@fortawesome/fontawesome-svg-core": "^1.2.28",
+		"@fortawesome/free-solid-svg-icons": "^5.13.0",
+		"@fortawesome/react-fontawesome": "^0.1.9",
+		"@parcel/transformer-svg-react": "^2.0.0-nightly.1739",
+		"@parcel/transformer-svgo": "^2.0.0-nightly.1739",
+		"@storybook/addon-actions": "^5.3.17",
```

Comment:
Given that we're going to separate the app from components again in the future and since components is meant to be published,  I would stick all devDependencies to the project root, keeping only things external to the build process here

Reply:
In general, I agree, but I would say that what goes into the root `package.json` should be determined by what will be reused across packages after the split, rather than what is external to the build process.

I will try to make to clean it up in a way that makes sense.

Reply:
Tried my best in 0fcbed9e3a61588cb8ab10868bd5b27439cc0b01, let me know if you're happy with the dependencies now

Reply:
LGTM!